### PR TITLE
Feature/step list

### PR DESCRIPTION
**Pull request by KhinThiriSoe**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -29,14 +29,12 @@ kapt {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
```
Comment (AungThiha) [inline app/build.gradle:32]: @KhinThiriSoe what's the difference?

Comment (KhinThiriSoe) [reply app/build.gradle:32]: different sequence :D sorry

Comment (KhinThiriSoe) [reply app/build.gradle:32]: It happens when I merged I guess

File: app/src/main/res/drawable-v24/default_indicator.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item>
+    <shape
+        android:innerRadius="0dp"
+        android:shape="ring"
+        android:thickness="8dp"
+        android:useLevel="false">
+      <solid android:color="@android:color/darker_gray"/>
```
Comment (AungThiha) [inline app/src/main/res/drawable-v24/default_indicator.xml]: @KhinThiriSoe put this color into color resource. the dimens are fine here.

File: app/src/main/res/layout-v21/fragment_step_video.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+    <com.google.android.exoplayer2.ui.PlayerView
+            android:id="@+id/ep_video_view"
+            android:layout_width="match_parent"
+            android:layout_marginBottom="@dimen/standard_margin"
+            android:layout_height="230dp"/>
```
Comment (AungThiha) [inline app/src/main/res/layout-v21/fragment_step_video.xml]: @KhinThiriSoe don't use literal value here. Use dimen resource

File: app/src/main/res/layout-v21/fragment_step_video.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+    <TextView
+            android:id="@+id/txt_short_description"
+            android:padding="16dp"
+            android:textSize="20sp"
+            android:textColor="@android:color/black"
```
Comment (AungThiha) [inline app/src/main/res/layout-v21/fragment_step_video.xml]: @KhinThiriSoe lateral values here again. Use dimen resource

**Comment by AungThiha** [comment]
@KhinThiriSoe check all your layout file for literal values and fix them

File: app/src/main/res/layout-v21/fragment_step_video.xml (hunk 4 above)
Comment (KhinThiriSoe) [reply app/src/main/res/layout-v21/fragment_step_video.xml]: fixed them all :)

File: app/src/main/res/layout/activity_steps.xml [hunk 5]
```diff
@@ -1,23 +1,29 @@
...
-            android:padding="@dimen/standard_padding"
+    <com.google.android.material.tabs.TabLayout
+            android:id="@+id/tabLayout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"/>
+            android:layout_height="wrap_content"
+            android:layout_alignBottom="@id/viewPager"
+            app:tabBackground="@drawable/tab_selector"
+            app:tabGravity="center"
+            app:tabIndicatorHeight="0dp"/>
```
Comment (AungThiha) [inline app/src/main/res/layout/activity_steps.xml]: @KhinThiriSoe tabindicatorHeight

File: app/src/main/res/layout/list_steps.xml [hunk 6]
```diff
@@ -1,20 +1,22 @@
...
+        xmlns:app="http://schemas.android.com/apk/res-auto" android:layout_width="match_parent"
         android:orientation="horizontal"
+        android:padding="@dimen/standard_margin"
         android:layout_height="wrap_content">
 
     <TextView
             android:id="@+id/txt_short_description"
             android:padding="@dimen/standard_padding"
             android:layout_width="0dp"
+            android:textSize="16sp"
```
Comment (AungThiha) [inline app/src/main/res/layout/list_steps.xml]: @KhinThiriSoe textSize

Comment (KhinThiriSoe) [reply app/src/main/res/layout/list_steps.xml]: Screen Shot 2019-03-12 at 1 04 52 PM

Comment (AungThiha) [reply app/src/main/res/layout/list_steps.xml]: @KhinThiriSoe that warning is for layout_width. I'm talking about textSize

Comment (KhinThiriSoe) [reply app/src/main/res/layout/list_steps.xml]: app:tabIndicatorHeight = 0dp I was wrong writing comment

Comment (AungThiha) [reply app/src/main/res/layout/list_steps.xml]: @KhinThiriSoe no, you're still wrong. the photo you added is for this `txt_short_description` and it's for `layout_width`

Let's talk about `app:tabIndicatorHeight` in a different thread

File: app/src/main/res/layout/activity_steps.xml (hunk 5 above)
Comment (KhinThiriSoe) [reply app/src/main/res/layout/activity_steps.xml]: fixed it

File: app/src/main/res/layout/list_steps.xml (hunk 6 above)
Comment (KhinThiriSoe) [reply app/src/main/res/layout/list_steps.xml]: done :)

**Pull request merged**